playbooks/install.py

```diff
@@ -12,7 +13,7 @@ def install_bench(args):
 	success = run_os_command({
 		'apt-get': [
 			'sudo apt-get update',
-			'sudo apt-get install -y git build-essential python-setuptools python-dev'
+			'sudo apt-get install -y git build-essential python-setuptools python-dev libssl-dev libffi-dev'
```

Comment:
@anandpdoshi Faced problems while testing the script on Ubuntu 14.04 while installing ansible which depends on cryptography which in turn has libssl-dev and libffi-dev in dependencies.